src/domain/platform/admin/templates/InnovationTemplates/UpdateInnovationFlow.tsx

```diff
@@ -37,6 +38,15 @@ const UpdateInnovationFlow: FC<EditLifecycleProps> = ({
   const innovationFlowTemplate = innovationFlowTemplates?.find(
     template => lifecycle && lifecycle.templateName && template.definition.includes(lifecycle.templateName)
   );
+  const { data } = useInnovationFlowAuthorizationQuery({
+    variables: { innovationFlowId: entityId! },
+    skip: !entityId,
+  });
+  const myPrivileges = data?.lookup.innovationFlow?.authorization?.myPrivileges;
+
+  const privileges = {
+    canUpdate: (myPrivileges ?? []).some(p => p === AuthorizationPrivilege.UpdateInnovationFlow),
```

Comment:
I know it's everywhere, probably you copy pasted this `some` from somewhere, but this can be simplified to `includes(AuthorizationPrivilege.UpdateInnovationFlow)` and we are tending to use that more